auto.def

```diff
@@ -694,8 +696,13 @@ apply {{ncurses_prefix} {
 
 ###############################################################################
 # Iconv - try to mimic AM_ICONV by preferring an installed libiconv
-if {![check-iconv $prefix]} {
-  user-error "Unable to find iconv()"
+if {[opt-val with-iconv] ne {}} {
+  set iconv_include [opt-val with-iconv $prefix]
+  define-append CFLAGS -I$iconv_include/include
```

Comment:
I think you also want to link from the same location

Reply:
I thought about that, but the @roccoblues's homebrew PR didn't include it
https://github.com/Homebrew/homebrew-core/pull/152445/files